src/ApplicationRoutes.test.tsx

```diff
@@ -42,21 +42,21 @@ describe('ApplicationRoutes', () => {
 
   it('should show token fornyet', () => {
     act(() => {
-      render(makeRoute('/token-fornyet'), container);
+      render(makeRoute('/nb/token-fornyet'), container);
```

Comment:
Ble det no eller nb? Og burde det være gjennomført? Tenker på Slack diskusjonen https://nav-it.slack.com/archives/C020NAK58N5/p1620115828017800